src/main/java/de/retest/recheck/printer/MetadataDifferencePrinter.java

```diff
@@ -12,7 +12,9 @@
 	@Override
 	public String toString( final MetadataDifference difference, final String indent ) {
 		final String prefix = "Metadata Differences:";
-		return indent + prefix + printDifferences( difference, indent + "\t" );
+		final String note = "\n\t  Please note that Metadata Differences do not affect the result and are not included"
```

Comment:
```suggestion
		final String note = "\n\t  Please note that metadata differences do not affect the result and are not included"
```

Or just "these differences" since we already say "Metadata Differences" above (which I also think should be "Metadata differences).